src/signs/ui/search/MesZLContent.tsx

```diff
@@ -85,7 +87,7 @@ export default function MesZlContent({
         <div
           dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(mesZlBody) }}
         />
-        {mesZl.split('\n').length > 7 && (
+        {mesZl.split('\n').length > 7 && cutOff !== -1 && (
```

Comment:
What's special about 7 and -1? Shouldn't the check be if the length is larger than cut off?

Reply:
It should check if the length is larger than the cut off. But passing -1 will not cut it off at all.